SYCL/ESIMD/api/functional/ctors/ctor_fill_core.cpp

```diff
@@ -144,7 +150,7 @@ int main(int, char **) {
       }
     }
   }
-
+#endif
```

Comment:
```suggestion
#endif // FULL_TYPE_COVERAGE
```

Reply:
I've applied this comment [here](https://github.com/intel/llvm-test-suite/pull/832/commits/e162bda82d57be4839dc67b58c109b6f7cf9b438#diff-6df0727accdf4d3a754136946c897c835113f6e13a4439c34444a0afed9eb945R153)